### ISSUE TITLE
accessibility?

### ISSUE THREAD

**Issue by kappesante** [responsively-org/responsively-app#1075]
# 🚀 Feature Request

### 📝 Description

Missing accessibility checks

### ✨ Describe the solution you'd like

Implement some accessibility checks for contrast, link announcements, and so on.

### ✍️ Describe alternatives you've considered

Polypane is the actual alternative. Accessibility is the only feature i miss.

**Comment by DxVxDOD** [comment]
Hi could I work on this ?

**Comment by astuanax** [comment]
I would be available to work on this. 
We could use Alpha and/or axe-core

**Comment by astuanax** [comment]
How would it work?

- Add a button like the screenshot button
- Add a11Y stylesheet (or others)
- Run axe-core and use the output to add red borer around elements that fial the test
-  Add sidebar with axe-core results.

If someone could help out to test, that would be great.

**Comment by kappesante** [comment]
i’m super excited to see this taken in consideration. if only i was able to help writing an app! 

(the screeshot feature should be getting a little more love but i won’t press you guys, accessibility is way more important imho).

thanks a lot

**Comment by themohammadsa** [comment]
@astuanax please feel free to spin up a PR for this issue. Looking forward! 🚀

**Comment by astuanax** [comment]
I'll get started on this. 
I would add a couple of buttons to mimic different type of colour blindness. 

https://github.com/astuanax/color-blindness-emulation

**Comment by astuanax** [comment]
I have finished the first menu with specific color vision issues. 

I made a short video showing the functionality. 
https://github.com/responsively-org/responsively-app/assets/1730624/9986dc57-193a-4783-b093-aaed16611691

Next week I will clean up and make a first PR, 
but I would really love someone to test this features before pushing the PR.

Please let me know who is able to help out with user testing. 

Thx already!

**Comment by manojVivek** [comment]
@astuanax This demo looks great, please let us know when you have a PR and I can help to test it.

**Comment by astuanax** [comment]
Added the pull request. 

Please let me know about the features and how you tested.

Code review is also very necessary.

**Comment by astuanax** [comment]
If you guys like it, I can continue adding the following items:

- Add a button like the screenshot button
- Add a11Y stylesheet (or others)  << there are a lot more of those. 
- Run axe-core and use the output to add red borer around elements that fial the test
- Add sidebar with axe-core results.

**Comment by astuanax** [comment]
@manojVivek 

I messed up the previous pr, but the new is clean and should be fine to merge after review and updated.

**Comment by kappesante** [comment]
things are moving! thanks to all!
